### PR TITLE
Feature/integrate insights media

### PR DESCRIPTION
**Pull request by Rijidij**

### CODE REVIEW

File: etna/insights/migrations/0005_auto_20210715_1404.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.1.8 on 2021-07-15 14:04
```
Comment (danbentley) [inline etna/insights/migrations/0005_auto_20210715_1404.py:1]: Would you mind merging these migrations?

`python manage.py migrate insights 004`

Delete the two migrations…

Recreate
`python manage.py makemigrations`

**Review by danbentley — COMMENTED** [review]
Thanks, looks good. 

As discussed, we're likely to use `wagtail.embeds.models.Embed` for embed-specific data (like name, description etc) but this is a starting point for making the content dynamic.

File: etna/media_embeds/blocks.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from wagtail.core import blocks
+
+
+class MediaEmbedAudioBlock(blocks.StructBlock):
+    heading = blocks.TextBlock(max_length=100)
+    datetime = blocks.DateTimeBlock(required=False)
```
[the remark this answers is not in the document]
Comment (Rijidij) [reply etna/media_embeds/blocks.py:7]: Agreed. I was following the lead from the template where this structure is used:
`<time datetime="2016-09-13">Tuesday 13 September 2016</time>`

File: etna/insights/migrations/0005_auto_20210715_1404.py (hunk 1 above)
Comment (Rijidij) [reply etna/insights/migrations/0005_auto_20210715_1404.py]: Done

**Comment by Rijidij** [comment]
Stale branch - superseded by DS-235

**Pull request closed without merging**